cpp/src/parquet/encoding_benchmark.cc

```diff
@@ -452,6 +452,134 @@ BENCHMARK(BM_ByteStreamSplitEncode_Float_Avx512)->Range(MIN_RANGE, MAX_RANGE);
 BENCHMARK(BM_ByteStreamSplitEncode_Double_Avx512)->Range(MIN_RANGE, MAX_RANGE);
 #endif
 
+template <typename DType, typename NumberGenerator>
+static void BM_DeltaBitPackingEncode(benchmark::State& state, NumberGenerator gen) {
+  using T = typename DType::c_type;
+  std::vector<T> values = gen(state.range(0));
+  std::vector<uint8_t> output(state.range(0) * sizeof(T), 0);
+  auto encoder = MakeTypedEncoder<DType>(Encoding::DELTA_BINARY_PACKED);
+  for (auto _ : state) {
+    encoder->Put(values.data(), static_cast<int>(values.size()));
+    encoder->FlushValues();
+  }
+  state.SetBytesProcessed(state.iterations() * values.size() * sizeof(T));
+}
+
+static void BM_DeltaBitPackingEncode_Int32_Equals(benchmark::State& state) {
+  BM_DeltaBitPackingEncode<Int32Type>(
+      state, [](size_t number) { return std::vector<int32_t>(number, 64); });
+}
+
+static void BM_DeltaBitPackingEncode_Int64_Equals(benchmark::State& state) {
```

Comment:
Do we expect different performance on this than on narrow data? Otherwise we might as well not bother.

Reply:
In fact I think no different when encoding currently. Maybe it will be a impact when compress is introduced. I will paste my testing result later.